pkg/controller/storagecluster/quickstarts_test.go

```diff
@@ -0,0 +1,79 @@
+package storagecluster
+
+import (
+	"context"
+	consolev1 "github.com/openshift/api/console/v1"
+	api "github.com/openshift/ocs-operator/pkg/apis/ocs/v1"
+	"github.com/stretchr/testify/assert"
+	"k8s.io/apimachinery/pkg/types"
+	"testing"
+)
+
+const (
+	quickstartTestDir = "../../../quickstarts/"
+)
+
+type Cases struct {
+	name      string
+	namespace string
+}
+
+func TestEnsureQuickStarts(t *testing.T) {
+	cases := []Cases{
```

Comment:
We need cases for File does not exist, incorrect File format etc.

Reply:
@umangachapagain We are checking for parsing and file errors in the ensureQuickStarts function. Will report err if any on line 34

Reply:
That would be too late and in Customer's environment.

I'd like tests to minimize the risk of us shipping a broken YAML.
That probably means adding a separate test that reads the YAMLs and parses it. Do you agree?

Reply:
alright.... i've added the test for checking the files